docgen/parser/model.go

```diff
@@ -71,6 +73,13 @@ type Method struct {
 	Operation *options.Operation
 }
 
+func (m *Method) HeaderID() string {
+	name := "method-"
+	name = name + m.Request.Verb + "-"
+	name = name + m.Name
```

Comment:
I wonder if we will ever need to have CSS ruleset that target specific API name. Maybe just `"method-" + m.Name` is enough for our purpose?

Reply:
we want to have different styles for different method types.

Reply:
Oh you maybe mean `name = name + m.Request.Verb + "-"` is enough?

That cannot be enough, the header IDs need to be unique in a page. This is not a class name, this is anchor ID

Reply:
oops, I understand it wrong in first review. Clear to me now.